tools/bin/make/airbyte_ci_check.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
I don't know why but I was told that `#!/usr/bin/env bash` is preferable.

Reply:
Yup good call out
![Uploading image.png…]()